.github/workflows/linux.yml

```diff
@@ -43,10 +43,10 @@ jobs:
         sudo update-alternatives --verbose --install /usr/bin/g++ g++ /usr/bin/g++-10 100
 
     - name: Update Compiler (Clang)
-      if: ${{ matrix.image == 'ubuntu-18.04' && matrix.compiler == 'clang' }}
+      if: ${{ matrix.image == 'ubuntu-20.04' && matrix.compiler == 'clang' }}
       run: |
         sudo apt update
-        sudo apt install clang-10 clang++-10
+        sudo apt install clang-10
```

Comment:
isn't clang++-10 a seperate package?

Reply:
I think they were combined? It says `clang++-10` isn't found.